src/log/SentryTarget.php

```diff
@@ -183,8 +183,8 @@ protected function getOptions(): array
     {
         $options = [
             'dsn'                  => $this->dsn ?: null,
-            'release'              => $this->release ?: null,
-            'environment'          => $this->environment ?: (App::env('CRAFT_ENVIRONMENT') ?: null),
+            'release'              => $this->release ?? App::env('CRAFT_CLOUD_BUILD_ID'),
+            'environment'          => $this->environment ?? App::env('CRAFT_ENVIRONMENT'),
```

Comment:
`App::env` returns `null` if it can't find anything, so it seemed more straightforward to use null coalescing here.